sql-prepared-plan-cache.md

```diff
@@ -17,6 +17,78 @@ When the execution plan cache is enabled, in the first execution every `Prepare`
 
 TiDB also supports execution plan caching for some non-`PREPARE` statements, similar to the `Prepare`/`Execute` statements. For more details, refer to [Non-prepared plan cache](/sql-non-prepared-plan-cache.md).
 
+## Session-level and instance-level Prepared Plan Cache
+
+TiDB supports both session-level plan cache and instance-level plan cache. Session-level plan cache maintains a separate cache for each session, while instance-level plan cache maintains a single shared cache across all sessions in an instance.
```

Comment:
We shall recommend customers to select instances plan cache, as as it cached more plans with given memory size. Otherwise, customers would have doubts on which one should be choosen.

Reply:
Any scenario we recommend session plan cache?